pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: dart_style
 # Note: See tool/grind.dart for how to bump the version.
-version: 1.0.10
+version: 1.0.10-dev
```

Comment:
This looks a little weird. I initially planned to publish 1.0.10 after the last commit, but decided to roll a few more changes into it. Hence the retroactive add of "-dev".